src/main/java/net/sf/jabref/gui/actions/ErrorConsoleAction.java

```diff
@@ -39,57 +36,14 @@
  */
 public class ErrorConsoleAction extends AbstractAction {
 
-    private final JFrame frame;
-    private final StreamEavesdropper streamEavesdropper;
-    private final Cache cache;
-
-    public ErrorConsoleAction(JFrame frame, StreamEavesdropper streamEavesdropper, Cache cache) {
-        super(Localization.menuTitle("Show error console"));
-        this.streamEavesdropper = streamEavesdropper;
-        this.cache = cache;
+    public ErrorConsoleAction(StreamEavesdropper streamEavesdropper, Cache cache) {
+        super(Localization.menuTitle("Show developer information"));
```

Comment:
"Developer information" is better then "error console", but maybe one would expect to get information about becoming a developer for JabRef (instead of information for us developer). What about a more general `Troubleshoot problems` or `View event log` or `Problem reporting`?